substrate/frame/bounties/src/lib.rs

```diff
@@ -221,9 +224,14 @@ pub mod pallet {
 		#[pallet::constant]
 		type BountyDepositPayoutDelay: Get<BlockNumberFor<Self, I>>;
 
-		/// Bounty duration in blocks.
+		/// The period that starts when a curator is approved, during which they must execute or
```

Comment:
As convention: One line of docs, one newline and then a larger paragraph with details. But content sounds good.

Rust analyzer will then use the first line as short description.